torchvision/models/vision_transformer.py

```diff
@@ -185,9 +216,10 @@ def __init__(
         self._init_weights()
 
     def _init_weights(self):
-        fan_in = self.conv_proj.in_channels * self.conv_proj.kernel_size[0] * self.conv_proj.kernel_size[1]
-        nn.init.trunc_normal_(self.conv_proj.weight, std=math.sqrt(1 / fan_in))
-        nn.init.zeros_(self.conv_proj.bias)
+        if self.conv_stem_layers is None:
+            fan_in = self.conv_proj.in_channels * self.conv_proj.kernel_size[0] * self.conv_proj.kernel_size[1]
+            nn.init.trunc_normal_(self.conv_proj.weight, std=math.sqrt(1 / fan_in))
+            nn.init.zeros_(self.conv_proj.bias)
```

Comment:
What about the initialization scheme for the stem layers case? Shouldn't we be initializing them as well or the default will do? cc @mannatsingh

Reply:
All the conv layers use default PyTorch init, except for the final 1x1 conv in the conv stem - https://github.com/facebookresearch/pycls/blob/main/pycls/models/vit.py#L275-L281.